Cargo.toml

```diff
@@ -15,3 +15,4 @@ categories = ["command-line-interface"]
 
 [dev-dependencies]
 log = "0.4"
+unicode-width = "0.1.5"
```

Comment:
`[dev-dependencies]` are not available for the "main" crate. You want `[dependencies]`.

Reply:
My bad. +1